uaclient/tests/test_status.py

```diff
@@ -855,7 +875,8 @@ def test_nonroot_user_uses_cache_and_updates_if_available(
             }
         )
 
-        assert expected_status == status.status(cfg=cfg)
+        assert expected_status != status.status(cfg=cfg)
```

Comment:
Shouldn't they be the same here ?

Reply:
No, because the `expected_status` is written to status-cache which non-root does not use.

Reply:
(Accidentally clicked the resolve/unresolve button three times haha)

If we don't expect the status to be `expected_status` then `expected_status` isn't really the expected status, is it?

Let's rename `expected_status` to `status_cache_value`

Reply:
@CalvoM tagging to make sure you see this comment ^^

Reply:
I deleted the test as we do not use `status_cache` for non-root user. @orndorffgrant